descriptors/ModuleDescriptor-template.json

```diff
@@ -161,6 +161,12 @@
     }
   ],
   "requires": [],
+  "optional": [
+    {
+      "id": "source-storage-records",
```

Comment:
This is needed as we make call to `GET /source-storage/records/{inventoryId}/formatted`

Reply:
maybe we should place it into ```requires``` instead?

Reply:
Done